lib/index.js

```diff
@@ -4,18 +4,6 @@ const Joi = require('@hapi/joi');
 
 const Monitor = require('./monitor');
 
-<<<<<<< HEAD
-const internals = {
-    validateOptions(options) {
```

Comment:
This is no longer necessary because of new `Joi.attempt(options, internals.schema)` use.